### ISSUE TITLE
need mechanism to pass custom DataTypeFactory

### ISSUE THREAD

**Issue by salashnik** [database-rider/database-rider#30]
for enums in postgres needed custom DataTypeFactory.
see http://dbunit.sourceforge.net/faq.html#postgresqlEnumTypes

also, may be provided any way for configuring any property of `DatabaseConfig`?

**Comment by rmpestano** [comment]
Hi @salashnik, can you try with dbrider 1.2.9-SNAPSHOT?

To provide your own datatype see example test here: https://github.com/TopalSergey/database-rider/blob/7ea6ba0ed46ad6618be4dad0bb9dcb32e292ba80/rider-core/src/test/java/com/github/database/rider/core/configuration/DBUnitConfigTest.java#L133

You can also pass the datatype factory in dbunit.yml, [see here](https://github.com/TopalSergey/database-rider/blob/7ea6ba0ed46ad6618be4dad0bb9dcb32e292ba80/rider-core/src/test/resources/config/sample-dbunit.yml#L11).

To use the snapshot you need to declare the following repository:

```
<repositories>
    <repository>
        <snapshots/>
        <id>snapshots</id>
        <name>libs-snapshot</name>
        <url>https://oss.sonatype.org/content/repositories/snapshots</url>
    </repository>
</repositories>
```

**Comment by essh** [comment]
Just a heads up that I tried this out recently via 1.2.9-SNAPSHOT and it worked well for me.

**Comment by rmpestano** [comment]
Hi @essh, thanks for the feedback. I've just released v1.2.9, I was waiting to add more features/fixes on this release but had no time and neither got contributions (indeed this issue was actually contributed).
